watchtower-plugin/src/main.rs

```diff
@@ -295,6 +296,38 @@ async fn get_tower_info(
     }
 }
 
+async fn ping(
+    plugin: Plugin<Arc<Mutex<WTClient>>>,
+    v: serde_json::Value,
+) -> Result<serde_json::Value, Error> {
+    let (tower_net_addr, proxy) = {
+        // Check if the tower_id is known to the plugin
+        let tower_id = TowerId::try_from(v).map_err(|e| anyhow!(e))?;
+        let state = plugin.state().lock().unwrap();
+        (
+            state
+                .towers
+                .get(&tower_id)
+                .ok_or(anyhow!("Unknown tower_id"))?
+                .net_addr
+                .clone(),
+            state.proxy.clone(),
+        )
+    };
+    let response = get_request(&tower_net_addr, Endpoint::Ping, &proxy)
+        .await
+        .map_err(to_cln_error)?;
+
+    if response.status().is_success() {
+        Ok(json!("Tower is reachable"))
+    } else {
+        Err(anyhow!(format!(
+            "Tower cannot be reached (Error code: {})",
```

Comment:
I've just checked this and turns out `status()` formats more than the code:

```
"Tower cannot be reached (Error code: 405 Method Not Allowed)"
```

So I'd change this to `"Tower cannot be reached (Error:  {})"`

Reply:
ohh ok